types/node-fetch/index.d.ts

```diff
@@ -14,6 +14,12 @@
 import { Agent } from "http";
 import { URLSearchParams } from "url";
 
+// `Event` and `AbortSignal` are defined here to prevent a dependency on a particular implementation
+// like the `abort-controller` package, and to avoid requiring the `dom` library
+// in `tsconfig.json`.
+import { AbortSignal } from "./externals";
+export * from "./externals";
```

Comment:
I don't think we should be exporting the assistive types here, as the consuming application will use a concrete implementation.

Reply:
My rationale here is that by exporting an interface, someone can force that their implementation `implements` it. Either way, not a big deal to me.